pages/auth/checkmail.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Layout from "../../components/Layout";
+import {useState} from 'react'
+import LoginErrorMessage from "../../components/utils/LoginErrorMessage";
+import {sendPasswordResetEmail } from "firebase/auth";
+import { auth } from "../../firebase.config";
+import Router from 'next/router'
+import { useCookies } from 'react-cookie';
+import { useTranslation } from "next-i18next";
+
+
+
+const checkmail = () => {
+  const [cookies, removeCookie] = useCookies();
+
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [successMessage, setSuccessMessage] = useState("hidden");
+  const [userMail,setUserMail]=useState(cookies.forgottenUser);
+
+  const { t } = useTranslation();
+  
+  const handleVerification = async() => {
+    sendPasswordResetEmail(auth, cookies.forgottenUser)
+    .then(() => {
+      console.log('mail sent')
+      setUserMail(cookies.forgottenUser)
```

Comment:
Please use query instead of cookies, as it is just client side and doesn't require this value in other pages. While using query, don't show the email id in the browser url.